src/controllers/users.js

```diff
@@ -2,28 +2,38 @@ import bcrypt from 'bcrypt';
 import jwt from 'jsonwebtoken';
 import environment from 'dotenv';
 import models from '../db/models';
+import validateLogin from '../validation/loginValidation';
 
 environment.config();
 
 export default class usersController {
   static async registerUser(req, res) {
-    try {
-      const {
-        firstName, lastName, gender, passportNumber, email, password,
-      } = req.body;
+    const {
+      firstName, lastName, gender, passportNumber, email, password,
+    } = req.body;
+    const salt = await bcrypt.genSalt(10);
+    const hashedPassword = await bcrypt.hash(password, salt);
 
-      const salt = await bcrypt.genSalt(10);
-      const hashedPassword = await bcrypt.hash(password, salt);
+    const newUser = await models.User.create({
+      firstName, lastName, gender, email, password: hashedPassword, passport: passportNumber,
+    });
+    const token = jwt.sign({
+      userId: newUser.id, email: newUser.email, firstName, lastName,
+    }, process.env.SECRETKEY);
+    res.status(201).json({ message: 'user successfully created', token });
+  }
 
-      const newUser = await models.User.create({
-        firstName, lastName, gender, email, password: hashedPassword, passport: passportNumber,
-      });
-      const token = jwt.sign({
-        userId: newUser.id, email: newUser.email, firstName, lastName,
-      }, process.env.SECRETKEY);
-      res.status(201).json({ message: 'user successfully created', token });
-    } catch (error) {
-      res.status(400).json({ error });
-    }
+  static async login(request, response) {
+    const { error, value } = validateLogin.validate(request.body);
+    if (error) return response.json(error.details[0].message);
+    const { email, password } = value;
+    const existUser = await models.User.findOne({ where: { email } });
+    if (existUser === null) return response.status(404).json({ status: 404, message: 'You are not registered' });
+    const passwordMatch = await bcrypt.compare(password, existUser.password);
+    if (!passwordMatch) return response.status(400).json({ status: 400, message: 'Wrong password' });
```

Comment:
not necessary to give a hint 
make it like `invalid user or password `

Reply:
I did it now you can check how it works